setup.cfg

```diff
@@ -33,6 +33,15 @@ zip_safe = False
 include_package_data = True
 install_requires =
     azure-monitor-opentelemetry-exporter == 1.0.0b6
+    opentelemetry-api == 1.11.1

```

Comment:
I'm not sure we can pin these versions because they would conflict with the versions installed by `azure-monitor-opentelemetry-exporter`. Why did we add these?

Reply:
Removed